jena-tdb/src/main/java/org/apache/jena/tdb/base/block/BlockMgrCache.java

```diff
@@ -58,10 +58,7 @@ private BlockMgrCache(int readSlots, int writeSlots, final BlockMgr blockMgr)
     {
```

Comment:
Wait-- that comment you made about a bug seems to be gone. I guess the question I'm asking is whether we want to include `TestCacheSimple` and fix the bug, or leave out `TestCacheSimple` and just mark the bug.